### DISCUSSION TITLE
How to save date in database using Date.now() in JS

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#1954] [Questions] [answered]
Hi I'm pretty in Supabase, I need to save date using Date.Now() and  for the type is timestamp.

Got this error,
```js
ERROR:  date/time field value out of range: "1489849402536"
```

Another error
![image](https://user-images.githubusercontent.com/9329005/121809825-fd2eed00-cc90-11eb-9ffe-3839e8199379.png)

**Comment by inian** [comment, marked as answer]
Are you using supabase-js @donPuerto? Can you try using `new Date().toISOString()` instead of `Date.now()`?

**Reply by donPuerto** [reply]
Yes. its working now. **new Date().toISOString()** and **timestampz** type
